src/main/java/org/sagebionetworks/web/server/servlet/InitSessionServlet.java

```diff
@@ -51,6 +51,27 @@ public void service(ServletRequest arg0, ServletResponse arg1) throws ServletExc
 		super.service(arg0, arg1);
 	}
 
+	public static Cookie getNewAccessTokenCookie(String accessToken, String requestScheme, String requestServerName) {
```

Comment:
Refactored code so the same thing can be used during SSO login to set the user_login_token cookie